src/main/java/org/truffleruby/core/numeric/IntegerNodes.java

```diff
@@ -1755,6 +1755,17 @@ protected int getLimit() {
 
     }
 
+    @Primitive(name = "mod_pow", lowerFixnum = { 0, 1, 2 })
```

Comment:
I'm not entirely sure if the `lowerFixnum` is necessary or what exactly it does, the documentation is kinda shady. My guess is that it will try to lower a possible `Bignum` to a `Fixnum` which are both deprecated in the latest version of Ruby, so I'm confused :confused:

Reply:
It tries to lower a Java `long` to a `int` (which both represent Ruby Fixnums), which can be useful if the `int` specialization is better.
In this case it's not useful as just using `long` is simpler and as efficient (`BigInteger.valueOf()` takes a `long`).